### DISCUSSION TITLE
Need some help after moving files in folder

### DISCUSSION THREAD

**Discussion by androa92** [stashapp/stash#2268] [Q&A] [answered]
Hi
I moved a lot of files/scenes from original folder structure, and declared the new folder in the library.
When a new scan is done, there is no update of the filepath in the SQL database.
I removed older folders from the library paths, and added new folder in the library paths.
Am I doing something wrong ?
Thanks for this amazing tool, by the way ! :)

**Comment by kermieisinthehouse** [comment, marked as answer]
This sounds like it should work, a scan is all it takes to update file paths. Were the files changed at all (transcoded, maybe?) when they were moved? Are you sure your new library paths are correct? What do the logs say while you're scanning?

Fyi, DO NOT run a clean operation until this is resolved, as it will delete your database entries if it can't find the files.

**Reply by androa92** [reply]
Thanks Kermie, I finally succeeded in re-scanning and regenerating the paths.
For an obscure reason, part of the folder was not scanned when I ask you for some help. And I was focusing on this part...
Re-naming it properly and re-scaning seems working well.
Thanks a lot for your quick answer, and godspeed for the future development of this great tool !
